src/components/ui/Flex.jsx

```diff
@@ -3,20 +3,35 @@ import classnames from 'classnames';
 import PropTypes from 'prop-types';
 
 const Flex = forwardRef(
-  ({ children, inline, className, justifyContent, alignItems = 'center', style, ...rest }, ref) => (
-    <div
-      className={classnames('flex', { 'flex--inline': inline }, className)}
-      ref={ref}
-      style={{
-        justifyContent,
-        alignItems,
-        ...style,
-      }}
-      {...rest}
-    >
-      {children}
-    </div>
-  )
+  (
+    {
+      as = 'div',
```

Comment:
The prettier reformatting makes it hard to see, but I've only added the capability to pass a custom HTML tag to this element, if we want something else than a `<div>`. This avoids some unnecessary element nesting.
This is a feature in the corresponding component of the DS, so it will ease the migration.